environment-wasm-host.yml

```diff
@@ -5,7 +5,7 @@ channels:
 dependencies:
   - nlohmann_json=3.11.3
   - xeus-lite
-  - xeus
+  - xeus=5.2.0
```

Comment:
I am not sure pinning xeus makes sense. Why should the native kernels use the latest version but the lite kernels use an older version.

Also the update in xeus is related to fixing this issue for lite kernels
https://github.com/compiler-research/xeus-cpp/issues/249 !